spec/src/main/asciidoc/01-introduction.adoc

```diff
@@ -0,0 +1,54 @@
+== Introduction
+
+=== Scope
+
+Jakarta Config ("Specification") is a Java API for working with configurations. It supports externalized configuration allowing applications to use different configurations for different environments (dev, test, prod), and allows reading data from different layered configuration sources such as property files, environment variables, etc.
+
+=== Goals
+
+- Specification must include a programmatic API allowing configuring and using configurations at runtime.
+- Specification must include annotations based API allowing compile-time configuration.
+- Specification must support converters and evaluate switching to Converters specification when it becomes available.
+- Specification must support object-mapping. Solution must respect Java methods visibility principles.
+- Specification must provide integration with CDI and optionally with  dependency injection frameworks.
+- Specification must distinguish empty strings and null values.
+- Specification must support profiles such as dev, test and prod.
+- Specification must support both mutable and immutable configuration sources.
```

Comment:
It would seem we have agreed otherwise: https://github.com/eclipse-ee4j/config/discussions/109